core/base/distanceField/DistanceField.h

```diff
@@ -142,11 +146,9 @@ int ttk::DistanceField::execute() const {
     std::stringstream msg;
     msg << "[DistanceField] Data-set (" << vertexNumber_
         << " points) processed in " << t.getElapsedTime() << " s. ("
-        << threadNumber_ << " thread(s))." << std::endl;
-    dMsg(std::cout, msg.str(), timeMsg);
+        << threadNumber_ << " thread(s)." << std::endl;
+    printMsg(msg.str(), ttk::Debug::debugPriority::timeMsg);
```

Comment:
you can have a look at this example (line 123) to see how to report timing:
https://github.com/topology-tool-kit/ttk/blob/dev/core/base/helloWorld/HelloWorld.h